src/components/Router/StepComponentMap.ts

```diff
@@ -265,9 +265,14 @@ const buildDocumentComponents = (
     countryCode,
     documentType
   )
+  const hasMultipleDocumentsWithUnsupportedCountry =
+    !hasOnePreselectedDocument && !supportedCountry
+  const hasCountryCodeNullOrBooleanDocumentTypeFlag =
```

Comment:
`hasCountryCodeNullOrBooleanDocumentTypeFlag` should probably be renamed to `hasCountryCodeOrDocumentTypeFlag` because in the code you are checking that `countryCode` is not `null`

Reply:
Good point. Will rename it.